ocs_ci/templates/ocp-deployment/rhcos_images.yaml

```diff
@@ -0,0 +1,12 @@
+4.3':
```

Comment:
@prsurve ,
Is it possible to have a jinja template for this as well and expand based on OCP version ? instead of storing for each and every version.

Reply:
I'm storing all the version because for every version kernel,intramfs, metal file path going to change

Reply:
I see a pattern 
```python
{OCP_MAJOR_VER}/{OCP_MINOR_VERSION}/rhcos-{OCP_MINOR_VERSION}-x86_64-<installer-initramfs|installer-kernel|metal>
```
Am I right ?

Reply:
yes that's possible to achive

Reply:
may be you can take it up in next PR

Reply:
will be covering this in next pr created an issue to track it #2645 2645